common.webpack.config.js

```diff
@@ -0,0 +1,46 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+
+module.exports = {
+
+    context: path.join(__dirname, 'src'),
+
+    entry: {
+        app: './index.jsx'
```

Comment:
please use `path.resolve` here not to rely on `/` separator

Reply:
Done